apps/costcontrol/test/unit/cost_control_test.js

```diff
@@ -498,4 +498,75 @@ suite('Cost Control Service Hub Suite >', function() {
       });
     }
   );
+
+  test(
+    'Global Data cache is refreshed when the resquest is not per App',
```

Comment:
nit: the global word seems to suggest a global scoped data cache. Instead, lets describe the test as `Querying data usage globally caches the result`.